Bixal/move.mil
# Adding www redirect and updating drupal core

## Checklist

I have…

      against my changes

## Summary of Changes

* Updates drupal core with the latest security release for symfony components.
* Adds www redirect for authoritative source

## Testing

* Navigate to the test environments bare domain. You should be redirect to www.
* Run normal tests after verifying version on status page. No updb is needed.
